internal/http/docker.go

```diff
@@ -0,0 +1,64 @@
+package http
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/hashicorp/go-retryablehttp"
+)
+
+type DockerRegistry struct {
```

Comment:
Hm, do we need a separate client? The endpoint you're calling belongs to the imagerunner service.

Reply:
Oh I'm not aware of that. Let me check.

Reply:
I'll merge it to `imagerunner` service.